data/entities.graphql

```diff
@@ -126,11 +127,20 @@ enum WorkspacePlan {
   Community
   Starter
   Pro
-  Enterprise
 }
 
 enum MembershipRole {
   Owner
   Admin
   Member
 }
+
+enum WorkspaceStatus {
```

Comment:
```suggestion
// https://stripe.com/docs/api/subscriptions/object#subscription_object-status
enum WorkspaceStatus {
```

Reply:
comments in graphql start with `#`

Reply:
OK. Thanks for having done the correct change instead 👍